ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/user-labels-tags/provision/cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision-chain.yaml

```diff
@@ -0,0 +1,16 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision
+  steps: 
+    - ref: gcp-provision-user-tags
```

Comment:
what about `gcp-provision-key-for-user-tags` or `gcp-provision-user-tags-key` or something like that?

Reply:
Removed the step according to Jianlin's suggestion, thanks!